.circleci/config.yml

```diff
@@ -60,36 +59,31 @@ commands:
           name: Building App
           no_output_timeout: 30m
           command: yarn build-travis
-  compress:
-    description: "Compress"
+  deploy:
+    description: "Deploy"
+    parameters:
+      target_branch:
+        type: string
     steps:
+      - checkout
+      - attach_workspace:
+          at: www
       - run:
-          name: "Compress"
+          name: Tag build
+          command: echo "<< parameters.target_branch >> $(date)" > www/version
+      - run:
+          name: Install and configure dependencies
           command: |
-            tar -cvf artifact.tar index.html nojs-smartcharts.css dist/*
+            sudo npm install -g gh-pages@2.0.1
+            git config user.email "mahboub.meng@gmail.com"
+            git config user.name "Mahboobeh-binary"
```

Comment:
Is it necessary to hardcode your `git user` and `email`?

Reply:
@afshinpaydar-binary we use machine user. it was for testing